2-mandatory/3-financial-times.js

```diff
@@ -23,19 +32,40 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let arr = [];
+
+  for (let moreClick of allArticleTitles) {
+    let choosen = "";
+    for (let letter of moreClick) {
+      if (!isNaN(letter) && letter !== " ") {
+        choosen = moreClick;
+        break;
+      }
+    }
+    if (choosen.length !== 0) {
+      arr.push(choosen);
+    }
+  }
+  return arr;
+
```

Comment:
regex is really useful for functions like this - this is a good site to learn more & have a play around https://regex101.com/
for the above you can do something like `(/\d/.test(stringYouWantToTest))` the `\d` checks for any digit, returning true if a digit is found